Cargo.toml

```diff
@@ -126,9 +127,11 @@ raw-window-handle = "0.6"
 rayon = "1"
 renderdoc-sys = "1.1.0"
 ron = "0.8"
-# rustc-hash 2.0 is a completely different hasher with different performance characteristics
+# NOTE: rustc-hash v2 is a completely different hasher with different performance characteristics
+# see discussion here (including with some other alternatives): https://github.com/gfx-rs/wgpu/issues/6999
+# (using default-features = false to support no-std build, avoiding any extra features that may require std::collections)
+rustc-hash = { version = "1", default-features = false }
```

Comment:
I tried to clarify the situation some more with these comments, @bushrat011899 / @cwfitzgerald please feel free to suggest any other updates that may be needed here.